src/pacer.js

```diff
@@ -564,9 +566,17 @@ let PACER = {
     //  - doc_number
     //  - att_number
     
+    let audio_string = $('td:contains(AUDIO)').text();
     let image_string = $('td:contains(Image)').text();
+    let transcript_string = $('td:contains(TRANSCRIPT:)').text();
```

Comment:
Some transcript files(such as those in `laeb`) use a `TRANSCRIPT:` prefix instead of a `Image` prefix.